public/js/main.js

```diff
@@ -15,18 +16,26 @@ const fnPrettify = function() {
     try {
         let inputJson = JSON.parse(inputEle.value);
         outputEle.innerHTML = `${JSON.stringify(inputJson, fnReplacer, 4)}`;
+        addLines(outputEle.innerHTML);
+
     } catch (e) {
         outputEle.innerHTML = "Invalid json";
     }
 }
 
-document.getElementById("idInputBox").onkeyup = function(e) {
-    var evt = e ? e : event;
-    if ((evt.keyCode && evt.keyCode != 13) || evt.which != 13)
-        return;
-    var elm = evt.target ? evt.target : evt.srcElement;
-    var lines = elm.value.split("\n");
-    for (var i = 0; i < lines.length; i++)
-        lines[i] = lines[i].replace(/(\d+\.\s|^)/, (i + 1) + ". ");
-    elm.value = lines.join("\n");
+outputEle.onscroll = function (e) {
+    verticalBar.scrollTop = this.scrollTop;
+};
+
+function addLines(e) {
+
+    let lines = e.split("\n");
+    for (var i = 0; i < lines.length; i++) {
+        lines[i] = "";
+        for(var j = 0; j < 6 - `${i+1}`.length; j++) {
+            lines[i] += `&nbsp;`;
```

Comment:
Why this is used ?

Reply:
To allow 1 line number (eg - 1 , 2 and so on) on a single line at a time...Otherwise it will lead to multiple no on a single line

Reply:
> To allow 1 line number (eg - 1 , 2 and so on) on a single line at a time...Otherwise it will lead to multiple no on a single line

Why this 6 - in for loop?...why 6 only???

Reply:
It depends on max length of the last no. Currently it is 6 so it can go for 999999 (length is 6 here).

Reply:
cool work bro